traits/trait_types.py

```diff
@@ -3336,6 +3338,144 @@ def as_ctrait(self):
         """
         return self.trait_maker.as_ctrait()
 
+# -------------------------------------------------------------------------------
```

Comment:
Minor style nit - we've been removing these headers as they were used by an old pre-sphinx auto-documenter and don't add anything to the code (it's clear that we are defining a None trait here).

Reply:
Removed them.